setup.py

```diff
@@ -77,6 +77,10 @@
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
```

Comment:
https://github.com/dropbox/dropbox-sdk-python/blob/f40e73fee9127ce0e3032724afc77a5f0e173f2b/setup.py#L36

I think something like `'setuptools >= 68.2.2'` should be added to the `setup_requires` list.
Without this I can't create a virtual env for this library on Python 3.12

See: https://docs.python.org/3/whatsnew/3.12.html

- [gh-95299](https://github.com/python/cpython/issues/95299): Do not pre-install setuptools in virtual environments created with [venv](https://docs.python.org/3/library/venv.html#module-venv). This means that distutils, setuptools, pkg_resources, and easy_install will no longer available by default; to access these run pip install setuptools in the [activated](https://docs.python.org/3/library/venv.html#venv-explanation) virtual environment.